src/views/technicalQualification.html

```diff
@@ -12,8 +12,13 @@
       {{> common/pageHeading pageHeading }}
 
       <p>
```

Comment:
Need to add IDs and add unit test

Reply:
I have added an id here and test for it's existence however I would argue that tests written to test all the hard coded text within the html within the page is exactly as required would render the tests brittle.